src/dune_pkg/local_package.ml

```diff
@@ -145,7 +145,26 @@ let of_package (t : Dune_lang.Package.t) =
   let name = Package.name t in
   match Package.original_opam_file t with
   | None ->
-    let dependencies = t |> Package.depends |> Dependency_formula.of_dependencies in
+    let depends =
+      match Package.depends t, Package.has_opam_file t with
+      | [], Exists true ->
+        let name = t |> Package.name |> Package_name.to_string in
+        let hints =
+          [ Pp.textf "Delete the package %S from `dune-project`" name
+          ; Pp.textf "Add a `depends` field in the `package` stanza of package %S" name
```

Comment:
Shouldn't you add a `or` somewhere? You need to do one or another, no?

Reply:
I think these hints are separate and in theory unordered (in practice they are probably still printed in order). Maybe the better solution would just be displaying one of the hints instead of two conflicting hints.

Reply:
Maybe, I don't have a strong opinion on this. If I chose one, I would go for the `depends` one, as this is a practice we try to enforce with Dune.

Reply:
Ok, I swapped the order and added an `or`.